server/conf/i18n/messages

```diff
@@ -152,23 +152,33 @@ button.select=Select
 button.startApp=Start an application
 # Button to view the applications belonging to an applicant.
 button.viewApplications=View applications
+# Button to view and add clients
+button.viewAndAddClients=View and add clients
 
 # Part of the confirmation banner which appears after a new client has been added to a community-based organization.
 content.clientCreated=Your new client {0} has been added to your client list. You can now begin submitting applications for this client.
+# The tooltip of the email input field
+content.emailTooltip=Add an email address for your client to receive status updates about their application automatically. Without an email, you or your community-based organization will be responsible for communicating updates to your client
```

Comment:
suggestion:  I think there should be a period at the end of the tooltip.

Reply:
period added.

Reply:
suggestion:  I meant at the end of the tooltip itself.  So:  "... will be responsible for communicating updates to your client."